style.css

```diff
@@ -36,6 +36,20 @@ body {
   background-color: #fff;
 }
 
+@media screen and (min-width: 600px) {
```

Comment:
- [ ] Great job using the `min-width`, however, the value for min-width property should be in range of 992px - 1100px, but not 600px, I can see that you have used 600px at several places, please let's use the range of 992px - 1100px.

Reply:
noted. I'll use that value